app/src/main/res/navigation/new_message_navigation.xml

```diff
@@ -19,10 +19,16 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/new_message_navigation"
-    app:startDestination="@id/newMessageFragment">
+    app:startDestination="@id/selectMailboxFragment">
```

Comment:
Is this how it will work by the end where we first start on selectMailboxFragment and then if there's no choice to make, then we go to newMessageFragment?